PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -21,7 +21,7 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return ((((float)numberCorrectAnswers / numberOfQuestions)) * 100) + "%";
```

Comment:
```suggestion
        return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Some unnecessary parenthesis here